docs/excel/custom-functions-naming.md

```diff
@@ -1,75 +1,101 @@
 ---
 title: Naming and localization for custom functions in Excel
 description: Learn requirements for names of Excel custom functions and how to localize custom functions.
-ms.date: 02/29/2024
+ms.date: 09/22/2025
 ms.localizationpriority: medium
 ---
+
 # Custom functions naming and localization
 
-This article describes guidelines and best practices for naming custom functions. It also shows how to localize custom function names to languages other than English.
+This article lists simple rules and tips for naming custom functions and shows how to localize them.
```

Comment:
This comes down to SEO, but if simple rules is more searched than guidelines, and tips more than best practices, it makes sense.